SmartDeviceLinkTests/SDLHapticRectViewController.h

```diff
@@ -0,0 +1,18 @@
+//
+//  SDLHapticRectViewController.h
+//  SmartDeviceLinkTests
+//
+//  Created by Nicole on 9/1/20.
+//  Copyright © 2020 smartdevicelink. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// Test view controller for testing the SDLFocusableItemLocator's preferredFocusEnvironments
+@interface SDLHapticRectViewController : UIViewController
```

Comment:
Should be named `TestHapticRectViewController` or something like that to make it clear that it's not a part of the lib.

Reply:
Fixed. Renamed.